lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -50,43 +50,77 @@ local function MakePropNoEffect(...)
 	return ret
 end
 
-function PropCore.CreateProp(self,model,pos,angles,freeze)
-	if(!util.IsValidModel(model) || !util.IsValidProp(model) || not PropCore.ValidSpawn() )then
-		return nil
+function PropCore.CreateProp(self,model,pos,angles,freeze,isVehicle)
+
+	if not PropCore.ValidSpawn() then return nil end
+	
+	if isVehicle then
+		if model == "" then model = "models/nova/airboat_seat.mdl" end
+		if model ~= "models/vehicles/prisoner_pod_inner.mdl" then angles = angles + Angle(0, -90, 0) end
```

Comment:
What I mean is, the user may intend the chair to be rotated a certain way, but this will cause their intended behavior to be rotated with an additional yaw of 90 degrees.